### PR TITLE
Jesse Hood - sprint completed

### PR DESCRIPTION
**Pull request by jessehood**

### CODE REVIEW

File: security/PracticalSecurity.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+I have already been using 2 factor auth for my accounts for a few years now. 
+I was using google authenticator because I was not aware that there were other options. 
+When my phone stopped working one day I had a difficult time trying to recover my accounts but eventually got all of them back except my coinbase account (I had to make a new one, and fortunately I didn't have any funds in the old account).
```
Comment (JCamp-12) [inline security/PracticalSecurity.md:3]: the coinbase thing scared me reading it until that no funds part.

File: security/ThreatModel.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+One of the bigger security threats right now is phone number spoofing scams.
```
Comment (JCamp-12) [inline security/ThreatModel.md:1]: elderly are always at risk ... but funny even that situation with a component of social hacking of clicking in the wrong place, downloading the wrong thing, or providing password or SS# details.  Almost always a combo

File: 1d-life/src/1d-life.js [hunk 3]
```diff
@@ -39,9 +39,15 @@
    * @param {*} x the current x coordinate in question
    */
   function getNewVal(lifeState, x) {
-    // !!! IMPLEMENT ME
+    const current = lifeState[x];
```
Comment (JCamp-12) [inline 1d-life/src/1d-life.js:42]: nicely implemented!  

this was another cool solution, just showing you something different.  Both look readable, you may like this 

    let left = lifeState[x - 1];
    let current = lifeState[x];
    let right = lifeState[x + 1];

    let allOrNothing = (left && current && right) || (!left && !current && !right);
    
    return !allOrNothing? 1: 0;

**Comment by JCamp-12** [comment]
great job Jesse, looks like you got these concepts well. Keep in mind, security wise you are a high value target now

**Comment by adfaris** [comment]
Jesse, your work clearly shows that you have a firm grasp of the topic. Your code logic is clear. As for Security great personal practice using 2-factor-auth, it is a great practice.

**Pull request closed without merging**